frontend/src/libs/maps/icon-name-to-plain-svg.tsx

```diff
@@ -8,6 +11,9 @@ const iconNameToIcon: Record<
 > = {
   'arrow': ArrowIcon,
   'plus': PlusIcon,
+  'chat-page': ChatPageIcon,
```

Comment:
Check guys' PR related to accomodation of current map

Reply:
Yeah, it's already merged so you just have to adapt your new icons